tidb-cloud/pause-or-resume-tidb-cluster.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Pause or Resume a TiDB Cluster
+summary: Learn how to pause or resume a TiDB cluster.
+---
+
+# Pause or Resume a TiDB Cluster
+
+TiDB Cloud allows you to easily pause and resume a cluster that is not in operation at all times.
+
+Comparing with backup and restore, pausing and resuming a cluster takes less time and keeps your cluster state information (including cluster version, cluster configurations, and TiDB user accounts).
+
+> **Note:**
+>
+> Currently, you cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). Once a Developer Tier cluster remains idle for 24 hours, the cluster hibernates automatically, and you can resume it by [connecting to your cluster](/tidb-cloud/connect-to-tidb-cluster.md) or clicking **Resume** for the cluster in the TiDB Cloud console.
```

Comment:
```suggestion
> **Note:**
>
> You cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). See [Automatic hibernation and resuming] for Developer Tier.
```

Reply:
How about putting this into Limitation section.

Reply:
This is not a limitation for Dedicated Tiers, but only for Developer Tiers. What about still keeping it as a note and adding similar info to https://docs.pingcap.com/tidbcloud/select-cluster-tier#developer-tier-special-terms-and-conditions?